### DISCUSSION TITLE
Collabora doesn't work behind NginxProxyManager

### DISCUSSION THREAD

**Discussion by Nackophilz** [nextcloud/all-in-one#1243] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Install NextCloud AIO
2. Enable Collabora in the AIO interface
3. Configure NginxProxy manager

### Expected behavior 
Configure nextcloud office, with the subdomain dedicated to nextcloud office with : https://office.nextcloud.com/

### Actual behavior 

The config above doesnt work, i can only acess to Collabora using my internal IP and port 
### Host OS 
Unraid / Linux

#### Nextcloud AIO version 
Nextcloud AIO v2.0.3
#### Current channel 
Latest 
#### Other valuable info 
Log doesnt provide any information to be honest, the log just doesnt move when i test it 

Here is my nginx proxy manager for collabora : 
Details :
![image](https://user-images.githubusercontent.com/61667226/194843973-a2f19cc0-5834-461c-aa81-e8d35237964c.png)

Custom location : 
![image](https://user-images.githubusercontent.com/61667226/194844056-85d98d84-7253-48e2-8bd4-1d3d81d21b67.png)

SSL : 
![image](https://user-images.githubusercontent.com/61667226/194844124-9413aad1-2f6f-4a31-8848-e73d0c21f2a5.png)

Advanced : 

 # static files
 location ^~ /browser {
   proxy_pass https://192.168.XX.XXX:9980;
   proxy_set_header Host $http_host;
 }

 # WOPI discovery URL
 location ^~ /hosting/discovery {
   proxy_pass https://192.168.XX.XXX:9980;
   proxy_set_header Host $http_host;
 }

 # Capabilities
 location ^~ /hosting/capabilities {
   proxy_pass https://192.168.XX.XXX:9980;
   proxy_set_header Host $http_host;
 }

 # main websocket
 location ~ ^/cool/(.*)/ws$ {
   proxy_pass https://192.168.XX.XXX:9980;
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection "Upgrade";
   proxy_set_header Host $http_host;
   proxy_read_timeout 36000s;
 }

 # download, presentation and image upload
 location ~ ^/(c|l)ool {
   proxy_pass https://192.168.XX.XXX:9980;
   proxy_set_header Host $http_host;
 }

 # Admin Console websocket
 location ^~ /cool/adminws {
   proxy_pass https://192.168.XX.XXX:9980;
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection "Upgrade";
   proxy_set_header Host $http_host;
   proxy_read_timeout 36000s;
 }

**Comment by szaimen** [comment]
Collabora is already included in AIO so you should not need to add an additional domain for collabora. It should work out of the box when you've set it up correctly by following https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

**Reply by Nackophilz** [reply]
I tried too, with without any configuration on npm (instead of the config for nextcloud)  

But this isn't work :/

**Comment by szaimen** [comment, marked as answer]
Can you restart the container from the aio interface and post the full logs of nextcloud-aio-nextcloud here? thanks!
